QModManager/API/SMLHelper/Initializer.cs

```diff
@@ -0,0 +1,48 @@
+﻿namespace QModManager.API.SMLHelper
+{
+    using Harmony;
+    using Patchers;
+    using QModManager.Utility;
+    using System;
+
+    internal static class Initializer
+    {
+        private static HarmonyInstance harmony { get => Patcher.harmony; }
+
+        internal static void PostPostInit()
+        {
+            Logger.Info($"Loading SMLHelper...");
+
+            try
+            {
+                Initialize();
+            }
+            catch (Exception e)
+            {
+                Logger.Error($"Caught an exception while trying to initialize SMLHelper");
+                Logger.Exception(e);
+            }
+        }
+
+        internal static void Initialize()
+        {
+            CustomFishPatcher.Patch(harmony);
+            TechTypePatcher.Patch(harmony);
+            CraftTreeTypePatcher.Patch(harmony);
+            CraftDataPatcher.Patch(harmony);
+            CraftTreePatcher.Patch(harmony);
+            DevConsolePatcher.Patch(harmony);
+            LanguagePatcher.Patch(harmony);
+            ResourcesPatcher.Patch(harmony);
+            PrefabDatabasePatcher.Patch(harmony);
```

Comment:
All those patches and all those logs...
This could be improved (i have an idea)

Reply:
It is true that this original structure is a relic of when SMLHelper exposed its patchers directly.  
Since the patchers themselves are no longer exposed publicly, there is an opportunity to combine and consolidate as needed.

Just keep readability in mind so that we don't end up with overly large files that are impossible to find anything in.

Reply:
I was thinking of attributes, but it would require an int parameter to be passed as the priority, and it would be a bit hard handling those priorities. Unless we can find a way of not needing the priority, this will not be a thing.